unmsmfisi-socialapplication/social_app
# [BE] set enviroment variables

## Description of Changes

In this pull request, we have made the following changes related to environment variable management for our application:

1. **Creation of .env.example**: A file named `.env.example` has been created which contains a list of all the environment variables required for the application to work properly. This file will serve as a reference for developers to configure their environment variables properly.

2. **Instructions to Configure Environment Variables**: Documentation has been added to explain to contributors and developers how to configure the environment variables for the application. To do this, they should duplicate the contents of the `.env.example` file into a new file named `.env`.

## Motivation

The motivation behind these changes is to provide a clear and consistent guideline for setting environment variables in our application. This will ease the configuration process for new developers and ensure that all necessary variables are defined properly.

## Impact

These changes will not affect the current functionality of the application, but will improve the documentation and configuration experience for developers. It will ensure that critical environment variables are present and configured correctly.

## Steps to Verify

1. Clone this repository on your local machine (if you haven't already done so).
2. Open the `.env.example` file and verify that it contains all the necessary environment variables with clear descriptions.
3. Create a new file named `.env` in the root of the project.
4. Copy the contents of `.env.example` into the new `.env` file.
5. Fill in the values of the environment variables in the `.env` file with the correct values for your development environment.
6. Run the application and verify that it works correctly with the environment variable settings.

## Additional Notes

Be sure to follow these instructions when setting environment variables in your development environment. If you have any questions or problems, feel free to comment on this pull request.
